examples/DynamicCantileverCase/dynamic_cantilever.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+from scipy.fft import fft, fftfreq
+from scipy.signal import find_peaks
+from elastica import *
+from dynamic_cantilever_helper import DynamicCantileverVibration
+
+
+def simulate_dynamic_cantilever_with(
+    density=2000,
+    n_elem=100,
+    final_time=300,
+    mode=0,
+    rendering_fps=30,  # For visualization
+):
```

Comment:
can we add some doc string here as well? For example, small explanation what is function doing, type of the inputs, related references.